### ISSUE TITLE
Should we split tcmods.php?

### ISSUE THREAD

**Issue by andig** [andig/moode#15]
- Move syscmds to settings
- Move audio template stuff to settings
- Move other commands to db/
- Move db/index.php to /commands.php

Seems like a sensible structure with split responsibilities?

**Comment by moodeaudio** [comment]
Yes, its code can b migrated to the other scripts, then it can be deleted. Same for tcmods-cs.php
